src/SCRIPTS/BF/ui.lua

```diff
@@ -302,22 +300,20 @@ function run_ui(event)
     -- process send queue
     mspProcessTxQ()
     -- navigation
-    if (event == userEvent.longPress.menu) then -- Taranis QX7 / X9
-        menuActive = 1
-        currentState = pageStatus.displayMenu
-    elseif userEvent.press.pageDown and (event == userEvent.longPress.enter) then -- Horus
+    if (event == EVT_VIRTUAL_ENTER_LONG) then
```

Comment:
Shouldn't this be `if (event == EVT_VIRTUAL_MENU_LONG) then` ?

Reply:
MENU_LONG cannot be used in telemetry scripts (triggers the “Reset ...” menu).